packer/netbsd.pkr.hcl

```diff
@@ -0,0 +1,137 @@
+      "p<wait2s>a<wait2s>c<wait2s>k<wait2s>e<wait2s>r<wait2s>",
+      "<enter><wait10s>",
+      "d<wait2s><enter><wait10s><enter><wait10s>",
+      "p<wait2s>a<wait2s>c<wait2s>k<wait2s>e<wait2s>r<wait2s>",
+      "<enter><wait10s>",
+      "p<wait2s>a<wait2s>c<wait2s>k<wait2s>e<wait2s>r<wait2s>",
+      "<enter><wait10s>",
+      "packer<enter><wait10s>",
+      "x<wait2s><enter><wait10s>",
+      "<enter><wait10s>",
+      "d<wait2s><enter><wait120s>",
+      "r<wait2s>o<wait2s>o<wait2s>t<wait2s>",
+      "<enter><wait10s>",
+      "p<wait2s>a<wait2s>c<wait2s>k<wait2s>e<wait2s>r<wait2s>",
+      "<enter><wait10s>",
+      "sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' /etc/ssh/sshd_config<enter><wait10s>",
+      "sed -i 's/#PasswordAuthentication yes/PasswordAuthentication yes/g' /etc/ssh/sshd_config<enter><wait10s>",
+      "sed -i 's/UsePam yes/UsePam no/g' /etc/ssh/sshd_config<enter><wait10s>",
+      "echo '/sbin/dhcpcd' > /etc/rc.local<enter><wait10s>",
+      "reboot<enter>"
+    ]
+
+  boot_wait               = "180s"
+  cpus                    = 4
+  disk_size               = 25600
+  memory                  = 4096
+  guest_additions_mode    = "disable"
+  guest_os_type           = "NetBSD_64"
+  headless                = true
+  http_directory          = "config"
+  iso_checksum            = "sha256:5f1bca14c4090122f31713dd86a926f63109dd6fb3c05f9b9b150a78acc8bc7e"
+  iso_urls                = [
+    "NetBSD-9.2-amd64.iso",
+    "https://cdn.netbsd.org/pub/NetBSD/NetBSD-9.2/images/NetBSD-9.2-amd64.iso"
+    ]
+  shutdown_command        = "/sbin/shutdown -p now"
+  ssh_username            = "root"
+  ssh_password            = "packer"
+  ssh_port                = 22
+  ssh_wait_timeout        = "900s"
+  vm_name                 = "netbsd92-gce.x86-64"
+}
+
+build {
+  name="netbsd"
+
+  sources = ["source.virtualbox-iso.vbox-gce-builder"]
+
+  provisioner "shell" {
+    script = "scripts/bsd/netbsd-prep-postgres.sh"
+  }
+
+  provisioner "shell" {
+    script = "scripts/bsd/netbsd-prep-gce.sh"
+  }
+
+  provisioner "file" {
+    source = "files/bsd/netbsd-rc.local.sh"
+    destination = "/etc/rc.local"
+  }
+
+  provisioner "file" {
+    source = "files/bsd/netbsd-rc.shutdown.sh"
+    destination = "/etc/rc.shutdown"
+  }
+
+  provisioner "shell" {
+    inline = ["chmod 744 /etc/rc.local && chmod 744 /etc/rc.shutdown"]
+  }
+
+  provisioner "shell" {
+    script = "scripts/bsd/netbsd-set-cron.sh"
+  }
+
+  post-processors {
+
+    post-processor "manifest" {
+      output     = "output/manifest.json"
+      strip_path = true
+    }
+
+    post-processor "shell-local" {
+      inline = [
+        "jq -r '.builds[] | (.name + \"/\" + .files[].name)' output/manifest.json | grep '\\.vmdk$' > output/disk-path", "if [ $(wc -l < output/disk-path) -gt 1 ]; then exit 1; fi", "qemu-img convert -f vmdk -O raw \"output-$(cat output/disk-path)\" output/disk.raw", "rm output/manifest.json output/disk-path"
```

Comment:
That's very hard to read. Perhaps you could format it more like the existing packer scripts? E.g.
```
    inline = [
      <<-SCRIPT
        cat /boot/loader.conf || true
        PAGER=cat freebsd-update fetch install
...
```